src/panel/direction/DirectionPanel.jsx

```diff
@@ -117,6 +118,8 @@ export default class DirectionPanel extends React.Component {
 
     if (this.props.selected !== prevProps.selected && this.state.routes.length > 0) {
       fire('set_main_route', { routeId: this.sanitizeSelected(), fitView: !isMobileDevice() });
+      const search = updateQueryString({ details: false });
+      window.app.navigateTo('routes/' + search, {}, { replace: false });
```

Comment:
It should also be restored to `false` when a new route is computed (for example by dragging the points) after the details of the current one have been displayed.

Reply:
Ok we will resolve history behavior in a separate PR as discussed